blogposts/2021/release-3.34

```diff
@@ -0,0 +1,43 @@
+    url: https://github.com/sourcegraph/sourcegraph/issues/27058
+    category: Search
+  - description: "Added logging of incoming Batch Changes webhooks to help site admins debug issues with webhook delivery. By default, sites without encryption will log webhooks for three days and sites with encryption will not log webhooks."
+    url: https://docs.sourcegraph.com/admin/config/batch_changes#incoming-webhooks
+    category: Admin
+  - description: "Batch Changes now requests `workflow` scope on GitHub personal access tokens to allow writing to the `.github` directory in repositories. If you have already configured a GitHub PAT for Batch Changes, we suggest adding this scope."
+    url: https://docs.sourcegraph.com/batch_changes/how-tos/configuring_credentials
+    category: Batch Changes
+  - description: "The search UI's repositories count as well as the GraphQL API's `search().repositories` and `search().repositoriesCount` have changed semantics from the set of searchable repositories to the set of repositories with matches. This results in increased search speed, and in a future release, we'll introduce separate fields for searchable repositories."
+    url: https://github.com/sourcegraph/sourcegraph/issues/26995
+    category: Search
+  - description: "Version contexts and `repogroup` filters (deprecated in v3.33) have been removed in favor of search contexts."
+    url: https://about.sourcegraph.com/blog/introducing-search-contexts/
+    category: Search
+  - description: "Saved search notifications (deprecated in v3.31.0) have been removed in favor of code monitoring."
+    url: https://docs.sourcegraph.com/code_monitoring
+    category: Search
+---
+
+Sourcegraph 3.34 is now available! Here are some highlights from this release:
+
+## Troubleshoot your searches with the "no results" page
+Searches that return no results can often result in a confusing experience for users. When this occurs, we now display helpful tips on how to use search, examples of how to solve common query issues, and links to tutorial videos addressing common use cases.
+<img style="max-width:calc(100% - 10px)" alt="Find implementations for Go" src="https://storage.googleapis.com/sourcegraph-assets/blog/3.34/no_results_page.png">
+
+## Quickly navigate your Go code with Find Implementations
+Navigating to implementations is now available for Go! If you have precise code intelligence enabled for your Go repositories, you can now click on Find Implementations to navigate to a symbol's interface definition. If you're at the interface definition itself, clicking on Find Implementations will show all the places where the interface is being implemented, allowing you to explore how it's being used by other users across repositories. It will also show you which interfaces a struct implements.
```

Comment:
```suggestion
If you have precise code intelligence enabled for your Go repositories, you can now click on "Find Implementations" to navigate to a symbol's interface definition. If you're at the interface definition itself, clicking on the same button will show all the places where the interface is being implemented, allowing you to explore how it's being used by other users across repositories. It can also show you which interfaces a struct implements.
```

Reply:
cc @macraig updated this to lead with the benefit.